aws-distro-opentelemetry-node-autoinstrumentation/package.json

```diff
@@ -22,7 +22,9 @@
   "repository": "aws-observability/aws-otel-js-instrumentation",
   "scripts": {
     "clean": "rimraf build/*",
-    "compile": "tsc -p .",
+    "compile:tsc": "tsc -p .",
```

Comment:
who will run "compile:tsc" if we shift "compile" to "compile:webpack" as code compiling entry point?